samples/oauth_php/VetsApiClaims/app/Http/Controllers/ClaimsController.php

```diff
@@ -18,6 +18,51 @@ public function index(Request $request)
             $response = $client->request('GET', $url);
             $claims = json_decode($response->getBody()->getContents(), true);
             return view('claims', ['claims' => $claims['data']]);
-        } 
+        }
+    }
+
+    public function poa_form(Request $request)
+    {
+        return view('poa_form', []);
+    }
+
+    public function poa_form_show(Request $request, $id)
+    {
+        $oauth_session = OauthSession::find($request->session()->get('id'));
+        $url = env('VETS_API_URL') . '/services/claims/v1/forms/2122/' . $id;
+        $client = new Client(['headers' => ['Authorization' => 'Bearer ' . $oauth_session->access_token]]);
+        $response = $client->request('GET', $url);
+        $claim = json_decode($response->getBody()->getContents(), true)['data'];
```

Comment:
How will this react if $response fails and/or is successful, but missing ['data'] - do these need to be in try/catches and validated before used - or handled gracefully so an error isn't rendered or shown?

Reply:
in this case it's retrieving the data that was just submitted so we _shouldn't_ see failures here, if so we got some bigger issues